assets/donate.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --gray: #798890;
+  --light-blue: #0f81e8;
+}
+
+.payments-titles {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin-bottom: 1em; /* 16px */
+  }
+
+  .payments-title {
+    border-bottom: 0.1875rem solid;  /* 3px */
+    border-bottom-color: #798890;
```

Comment:
nit: `var(--gray);` as well

Reply:
the duplicate properties here are all to provide support for browsers that may not support CSS variables: https://caniuse.com/css-variables. not sure if we want to support those browsers, but wanted to be careful just in case!